src/matrix.jl

```diff
@@ -49,10 +49,10 @@ Wrap an `AbstractMatrix` (`Matrix`, `Adjoint`, etc.) in a `MatrixTable`, which s
 Tables.jl interface. This allows accessing the matrix via `Tables.rows` and `Tables.columns`.
 An optional keyword argument iterator `header` can be passed which will be converted to a
 `Vector{Symbol}` to be used as the column names. Note that no copy of the `AbstractMatrix`
-is made, but `header` is always stored as freshly allocated object.
+is made, but unless the `header` is a `Vector{Symbol}`, a new copy is allocated.
```

Comment:
I think it is enough to write:
```suggestion
is made.
```
since above we say that `header` is converted to `Vector{Symbol}` which implies that if it is `Vector{Symbol}` no copy is made and otherwise there is a copy (this is how `convert` works, although internally we do not use `convert` we do essentially the same).